### ISSUE TITLE
[Feature Request] Support HParam logging of tuples

### ISSUE THREAD

**Issue by AlexPasqua** [DLR-RM/stable-baselines3#1096]
### 🚀 Feature

Add support for logging tuples on Tensorboard through `HParam`, in addition to ints, strings, tensors...

### Motivation

I needed to log the dimensions of the layers of a neural net I'm using, so for different experiments I have a different number of layers and also different numbers of neurons per layer.
I was representing this data in my software with a tuple: `(#neurons 1st layer, #neurons 2nd layer, ...)`

### Alternatives

Right now I'm casting the tuple to string and loggind that.
It's an ok workaround, but I thing it would be better to have the actual tuple of integers, especially if some manipulations of the logged data is needed.

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by araffin** [comment]
Hello,
looking at the code, this should be already supported, we don't transform what is passed to the metric dict. If not, then the issue probably comes from pytorch.
Another solution is to have a key per layer.

**Comment by AlexPasqua** [comment]
> Hello, looking at the code, this should be already supported, we don't transform what is passed to the metric dict. If not, then the issue probably comes from pytorch.

Yes, it looks like it originates from pytorch, sorry for the "useless" issue and thanks for the reply
